tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -89,6 +89,8 @@ class ContextDeviceMemory {
     device_memory_ = nullptr;
     VLOG(2) << "Device memory size for TensorRT engine " << device_memory_size;
     if (device_memory_size > 0) {
+      tensorflow::profiler::TraceMe activity(
+          "TFTRT_Allocator", tensorflow::profiler::TraceMeLevel::kInfo);
```

Comment:
Would it be better to put this inside TRTDeviceAllocator::allocate?

Reply:
If we put into TRTDeviceAllocator, then it would log any allocation where TRT is using our allocator, which we do not want. I think we can just delete this TraceMe log since in practice the allocator is very fast.

Reply:
Then let's delete this log. I can't think of a good reason why we want to log the place that make the call and not log the callee itself.

Reply:
Removed. Fixed